### PR TITLE
Changes for German language

### PR DESCRIPTION
**Pull request by Hakermann420**
I found an issue in the (german) registration email. (The App name was written as %!!(string=<AppName>)) There was an s missing at the end of a line.
I also translated the english rest in the docs/content/page/index.de-de.md into german

### CODE REVIEW

File: docs/content/page/index.de-de.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ Gitea ist ein [Gogs](http://gogs.io)-Fork.
...
-- 2 CPU cores and 1GB RAM is typically sufficient for small teams/projects.
-- Gitea should be run with a dedicated non-root system account on UNIX-type systems.
-   - Note: Gitea manages the `~/.ssh/authorized_keys` file. Running Gitea as a regular user could break that user's ability to log in.
-- [Git](https://git-scm.com/) version 1.7.2 or later is required. Version 1.9.0 or later is recommended. Also please note:
-   - Git [large file storage](https://git-lfs.github.com/) will be available if enabled when git >= 2.1.2.
-   - Git commit-graph rendering will be enabled automatically when git >= 2.18.
+- Ein Raspberry Pi 3 ist leistungsstark genug, um Gitea für kleine Belastungen laufen zu lassen.
+- 2 CPU Kerne und 1GB RAM sind für kleine Teams/Projekte ausreichend.
+- Gitea sollte unter einem seperaten nicht-root Account auf UNIX-Systemen ausgeführt werden.
+   - Bemerke: Gitea verwaltet die `~/.ssh/authorized_keys` Datei. Gitea unter einem normalen Benutzer auszuführen könnte dazu führen, dass dieser sich nicht mehr anmelden kann.
```
Comment (delvh) [inline docs/content/page/index.de-de.md]:
```suggestion
   - Achtung: Gitea verwaltet die `~/.ssh/authorized_keys` Datei. Gitea unter einem normalen Benutzer auszuführen könnte dazu führen, dass dieser sich nicht mehr anmelden kann.
```

File: docs/content/page/index.de-de.md [hunk 2]
```diff
@@ -22,13 +22,13 @@ Gitea ist ein [Gogs](http://gogs.io)-Fork.
...
+- [Git](https://git-scm.com/) Version 1.7.2 oder später wird benötigt. Version 1.9.0 oder später wird empfohlen. Bemerke außerdem:
```
Comment (delvh) [inline docs/content/page/index.de-de.md]:
```suggestion
- [Git](https://git-scm.com/) Version 1.7.2 oder später wird benötigt. Version 1.9.0 oder später wird empfohlen. Außerdem zu beachten:
```

File: docs/content/page/index.de-de.md [hunk 3]
```diff
@@ -22,13 +22,13 @@ Gitea ist ein [Gogs](http://gogs.io)-Fork.
...
+   - Git [large file storage](https://git-lfs.github.com/) wird verfügbar sein, wenn es aktiviert ist und git >= 2.1.2.
```
Comment (delvh) [inline docs/content/page/index.de-de.md]:
```suggestion
   - Wenn git >= 2.1.2. und [Git large file storage](https://git-lfs.github.com/) aktiviert ist, dann wird es auch in Gitea verwendbar sein.
```

File: docs/content/page/index.de-de.md [hunk 4]
```diff
@@ -22,13 +22,13 @@ Gitea ist ein [Gogs](http://gogs.io)-Fork.
...
+   - Git Commit-Graph rendern wird automatisch aktiviert wenn git >= 2.18.
```
Comment (delvh) [inline docs/content/page/index.de-de.md]:
```suggestion
   - Wenn git >= 2.18, dann wird das Rendern von Commit-Graphen automatisch aktiviert.
```

**Review by delvh — APPROVED** [review]

File: options/locale/locale_de-DE.ini [hunk 5]
```diff
@@ -326,7 +326,7 @@ hi_user_x=Hallo <b>%s</b>,
 
 activate_account=Bitte aktiviere dein Konto
 activate_account.title=%s, bitte aktiviere dein Konto
-activate_account.text_1=Hallo <b>%[1]s</b>, danke für deine Registrierung bei %[2]!
+activate_account.text_1=Hallo <b>%[1]s</b>, danke für deine Registrierung bei %[2]s!
```
Comment (KN4CK3R) [inline options/locale/locale_de-DE.ini]: This needs to be mirrored to Crowdin.

Comment (6543) [reply options/locale/locale_de-DE.ini]: APPROVED - will be in next crowdin update (on midnight)

Comment (6543) [inline options/locale/locale_de-DE.ini]:
```suggestion
activate_account.text_1=Hallo <b>%[1]s</b>, danke für deine Registrierung bei %[2]!
```

**Review by 6543 — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Pull request merged**